packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -66,34 +66,27 @@ export class InputGroup extends React.Component<HTMLInputProps & IInputGroupProp
             Classes.intentClass(intent),
             {
                 [Classes.DISABLED]: this.props.disabled,
+                [Classes.ACTIVE]: this.state.isActive,
```

Comment:
_oh I see_. still, this breaks the CSS API for this component pretty wholly.

Reply:
@giladgray stashing this and punting for now